Process.h

```diff
@@ -15,6 +15,7 @@ in the source distribution for its full text.
 #include "Object.h"
 #include "ProcessField.h"
 #include "RichString.h"
+#include "Row.h"
```

Comment:
The `Table_` symbol has been used in this file.
```suggestion
#include "Row.h"
#include "Table.h"
```

Reply:
If `ProcessList` was not used before, `Table` shouldn't be used now either …

Reply:
Thanks, I'll drop this include in the next update.